modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14999,6 +15001,7 @@ select-an-existing-list-or-add-a-list-to-be-displayed-in-this-portlet=Select an
 select-an-existing-one=Select an Existing One
 select-an-existing-set-or-create-a-new-one-to-save-your-page-template=Select an existing set or create a new one to save your page template.
 select-an-existing-specification=Select an Existing Specification
+select-an-object-definition=Select an object definition.
```

Comment:
@cgmonte Maybe this could be a title, based on the near language keys?

Reply:
Yes it is, and it was actually fixed by Brian in the last PR. Rebasing excluded this change in my local branch an title case is in use. Thanks